modules/claims_api/app/workers/claims_api/ews_updater.rb

```diff
@@ -10,19 +10,19 @@ module ClaimsApi
   class EwsUpdater
     include Sidekiq::Worker
     FILE_5103 = 'Y'
+    OMITTED_FIELDS = %w[contentions dvlpmt_items letters name status_messages station_profile stn_suspns_prfil].freeze
```

Comment:
these were provided via a message in camelcase, but our tooling uses snake so I just converted them. Unfortunately they aren't displayed if they don't exist on the claim, so I am trying to locate a few test claims to see at least a majority of these...